apps/shared/lib/validation/tableschema_validator.ex

```diff
@@ -42,14 +42,13 @@ defmodule Shared.Validation.TableSchemaValidator do
     end
   end
 
-  defp build_report(%{"report" => %{"stats" => stats, "tasks" => tasks}} = report) do
-    nb_errors = Map.fetch!(stats, "errors")
-
+  defp build_report(%{"report" => %{"tasks" => tasks}} = report) do
     if Enum.count(tasks) != 1 do
       raise "tasks should have a length of 1 for report #{report}"
     end
 
     raw_errors = hd(tasks)["errors"]
+    nb_errors = Enum.count(raw_errors)
```

Comment:
```suggestion
    # we count the errors on our side, because error count given by the report can be wrong
    # https://git.opendatafrance.net/validata/validata-core/-/issues/37
    nb_errors = Enum.count(raw_errors)
```